src/pages/challenges/index.tsx

```diff
@@ -65,7 +59,7 @@ const Challenges: FC = () => {
                     (isLoggedIn ? (
                         isLoading ? (
                             <p>Loading...</p>
-                        ) : error ? (
+                        ) : isError ? (
```

Comment:
can you please change to text like-- "Something Went Wrong, Please Contact Admin !"